src/plan/executor.go

```diff
@@ -174,7 +174,7 @@ func (e *Executor) saveBranch(branch *state.BranchState, result *diff.Result) {
 
 func (e *Executor) saveConfig(config *state.ConfigState, result *diff.Result) {
 	pool := e.getPoolFor(config.Level())
-	if config.Local.Id == "" {
+	if config.Remote == nil {
```

Comment:
V kode bola uz nejaka priprava na vytvaranie noveho configu/row pri `push` ale bolo to potrebne dotiahnut.

`config.Remote == nil ` tj. config existuje iba lokalne.